gcore/resource_gcore_dns_zone_record_test.go

```diff
@@ -27,7 +27,7 @@ resource "%s" "%s" {
   zone = "%s"
   domain = "%s"
   type = "TXT"
-  ttl = 10
+  ttl = 210
```

Comment:
120 ?

Reply:
210 intentional, to make it different from the other test, but can be 120 if you want
since 120 is the lowest minimum allowed for free plan